ui/src/app/utils/redux.js

```diff
@@ -61,10 +65,83 @@ export const createStandardActions = name => {
       }
     }
   }));
+  action.delete.notify = createAction(`DELETE_${name.toUpperCase()}_NOTIFY`);
 
   action.cleanup = createAction(`CLEANUP_${name.toUpperCase()}`);
 
-  createAsyncActions(name, action);
+  createStandardAsyncActions(name, action);
 
   return action;
 };
+
+export const setStart = state => {
```

Comment:
Do these functions need to be separated out? I feel like `createStandardReducer` should give the full story of how an object in state can change, without any further abstractions. And since `fetch` doesn't follow the pattern it's a little confusing.